example/.env.example

```diff
@@ -14,6 +14,9 @@ STRIPE_SECRET_KEY_AU=sk_test
 STRIPE_PUBLISHABLE_KEY_WECHAT=pk_test
 STRIPE_SECRET_KEY_WECHAT=sk_test
 
+STRIPE_PUBLISHABLE_KEY_FINANCIAL_CONNECTIONS=pk_test
+STRIPE_SECRET_KEY_FINANCIAL_CONNECTIONS=sk_test
```

Comment:
We're using a separate merchant for Financial Connections, so that we can guarantee a native flow (and not a webview).